pkg/cmd/server/origin/ensure.go

```diff
@@ -179,6 +183,39 @@ func (c *MasterConfig) ensureComponentAuthorizationRules() {
 			glog.Errorf("Error creating bootstrap policy: %v", err)
 		}
 
+		// these are namespaced, so we can't reconcile them.  Just try to put them in until we work against rbac
```

Comment:
> Er so which role / binding are we missing exactly?

Several in kube-system.  One of which is related to auto-config of aggregated api servers.